cmd/config/subcommand/sandbox/sandbox_config.go

```diff
@@ -44,6 +44,9 @@ type Config struct {
 	// from there.
 	Image string `json:"image" pflag:",Optional. Provide a fully qualified path to a Flyte compliant docker image."`
 
+	// Default value false represents that flytectl will not use latest pre release if exist
+	Prerelease bool `json:"pre" pflag:",Optional. Pre release Version of flyte will be used for sandbox."`
```

Comment:
nit : can we also mention here which are other flags if passed in and there behavior eg : version or image